### PR TITLE
fix(shape_estimation): preserve irregular large size vehicle detected by camera_lidar_fusion as unknown

### PR DESCRIPTION
**Pull request by badai-nguyen**
## Description

- Current shape_estimation limits the size of known objects (vehicles) below some threshold and it will ignore some supper large trailer or crane.
- Adjust the size threshold to cover such irregular size vehicles might cause the degredation of estimation performance overall.
- This PR to reverse such irregular size vehicle as Unknown object.

## Related Link

[TIER IV INTERNAL LINK](https://tier4.atlassian.net/browse/RT1-5644)

## Tests performed
Confirmed by logging_simulator with internal rosbag that the camera_lidar_fusion result of left-side crane detection (`/perception/object_recognition/detection/clustering/camera_lidar_fusion/objects`) has been improved.
Screenshot from 2024-03-12 17-18-20

 - Before change (Green): crane was lost some times
 - After change (Purple): more stable
 - [Screencast from 2024年03月12日 19時51分17秒.webm](https://github.com/autowarefoundation/autoware.universe/assets/94814556/64d2d08f-adb8-401d-b577-52674e19a06e)

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by yukkysaito — APPROVED** [review]
LGTM

**Comment by YoshiRi** [comment]
Please wait for this to be merged.

**Review by technolojin — APPROVED** [review]
I confirmed the issue is resolved, via the issue data replay and recompute.
Thank you.

**Review by YoshiRi — CHANGES_REQUESTED** [review]
Sorry, could you change variable name?

File: perception/shape_estimation/config/shape_estimation.param.yaml
Comment (badai-nguyen) [reply perception/shape_estimation/config/shape_estimation.param.yaml]: @YoshiRi Thank you for you comment, I fixed at e77d669b6547910c71040b4d93bd62c3f441fd3e

**Review by YoshiRi — APPROVED** [review]
LGTM. Please change autoware_launch before merge this.

**Comment by badai-nguyen** [comment]
> LGTM. Please change autoware_launch before merge this.

@YoshiRi Since no shape_estimation releated parameters registered in autoware_launch so It's unneccessary.
BTW, I also tested on CI and confirmed there is no degradation. https://evaluation.tier4.jp/evaluation/reports/0df16ed5-ce10-5bd7-acb5-b8ee07b6e378?project_id=prd_jt
So I will merge this PR.

**Pull request merged**
